<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>751</width>
      <height>341</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="RETR">
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>-250</x>
        <y>-20</y>
        <width>581</width>
        <height>400</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="servercontents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>579</width>
         <height>398</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>391</x>
        <y>181</y>
        <width>57</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Filename</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputfilename">
      <property name="geometry">
       <rect>
        <x>455</x>
        <y>181</y>
        <width>159</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="download">
      <property name="geometry">
       <rect>
        <x>634</x>
        <y>181</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>DownLoad</string>
      </property>
     </widget>
     <widget class="QLabel" name="serverinfo">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>280</y>
        <width>64</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>List of files in server:</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>581</width>
        <height>400</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="clientcontents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>579</width>
         <height>398</height>
        </rect>
       </property>
       <widget class="QTreeView" name="treeView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>551</width>
          <height>411</height>
         </rect>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <attribute name="headerCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>300</number>
        </attribute>
        <attribute name="headerMinimumSectionSize">
         <number>0</number>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="clientinfo">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>250</y>
        <width>64</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;List of files in client:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>57</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Filename</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputfilename_2">
      <property name="geometry">
       <rect>
        <x>504</x>
        <y>10</y>
        <width>159</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="upload">
      <property name="geometry">
       <rect>
        <x>683</x>
        <y>10</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>UpLoad</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refresh">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>60</y>
        <width>86</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>refresh</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Login">
     <widget class="QLineEdit" name="username">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>160</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>cn2018</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>210</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ftp</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>210</y>
        <width>64</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Login_btn">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>210</y>
        <width>86</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ip_input">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>250</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>166.111.80.237</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="port_input">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>290</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>8279</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>64</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>IP Address</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>290</y>
        <width>64</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PORT</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
